azure-pipelines.yml

```diff
@@ -1,162 +1,888 @@
+              - task: CmdLine@2
+                displayName: Install app hosting dependencies (skip Cypress install)
+                condition: eq(variables.CYPRESS_CACHE_RESTORED, 'true')
+                inputs:
+                  script: |
+                    CYPRESS_INSTALL_BINARY=0 pnpm install --frozen-lockfile
+                  workingDirectory: '$(AppHostingSdkProjectDirectory)'
+              - task: CmdLine@2
+                displayName: Install app hosting dependencies (include Cypress install)
+                condition: eq(variables.CYPRESS_CACHE_RESTORED, 'false')
+                inputs:
+                  script: |
+                    rm -rf $(pnpm store path)
+                    pnpm install --frozen-lockfile
+                  workingDirectory: '$(AppHostingSdkProjectDirectory)'
+              - task: CmdLine@2
+                displayName: 'Build app hosting sdk'
+                inputs:
+                  script: |
+                    pnpm build
+                  workingDirectory: '$(AppHostingSdkProjectDirectory)'
+              - task: CmdLine@2
+                displayName: 'Install client sdk dependencies'
+                inputs:
+                  script: |
+                    pnpm install
+                  workingDirectory: '$(ClientSdkProjectDirectory)'
+              - task: CmdLine@2
+                displayName: 'Build client sdk'
+                inputs:
+                  script: |
+                    pnpm build-force-blazor
+                  workingDirectory: '$(ClientSdkProjectDirectory)'
+              - task: CmdLine@2
+                displayName: 'Configure host machine'
+                inputs:
+                  script: "sudo chmod -R 755 ./ \npnpm run setup\n"
+                  workingDirectory: '$(AppHostingSdkProjectDirectory)'
+              - task: CmdLine@2
+                displayName: 'Build Test App CDN'
+                inputs:
+                  script: |
+                    pnpm build-test-app-CDN
+                  workingDirectory: '$(ClientSdkProjectDirectory)'
+              - task: Bash@3
+                displayName: 'Run E2E integration tests with local script tag on latest cdn bundles'
+                condition: succeeded()
+                enabled: true
+                inputs:
+                  targetType: inline
+                  script: 'node tools/cli/runAppsWithE2ETests.js --reportFileName=e2e-tests-report-cdn-script-tag --envType=cdnScriptTag'
+                  workingDirectory: '$(AppHostingSdkProjectDirectory)'
+              - task: PublishTestResults@2
+                inputs:
+                  testResultsFormat: 'JUnit'
+                  testResultsFiles: '**/e2e-tests-report*.xml'
+                  testRunTitle: 'E2E Tests - CDN'
+                  mergeTestResults: true
+                condition: succeededOrFailed()
+          - job: E2ETestAndroid
+            displayName: 'E2E Test - Android'
+            pool:
```

Comment:
these pools used to be internal pools and not the published ones. Not sure why. For now have switched to the official pools, let's see what happens